python/cuml/ts/batched_arima.pyx

```diff
@@ -0,0 +1,42 @@
+import numpy as np
+
```

Comment:
Probably `arima.py` should be a cython file (`arima.pyx`) and this part of that single cython file